### DISCUSSION TITLE
Accept collections in plaster parameters

### DISCUSSION THREAD

**Discussion by chreestopher** [PowerShellOrg/Plaster#380] [Ideas]
I think a great feature would be the ability for plaster parameters to accept collection with similar behavior to  how powershell parameters that are typecast to a collection behave:

    Enter thing[0]: something
    Enter thing[1]: someOtherThing
    Enter thing[2]:

**Comment by bwright86** [comment]
+1

I would like to see this as well.

**Comment by bwright86** [comment]
To give examples... When multiple values need to be passed in for a parameter, so they can be joined together into a string, but different templates require them to be concatenated in in different ways.

So something like, `<%$($PLASTER_PARAM_List1 -join ",\n")%>` would be useful. It is similar to multichoice, but would allow open ended input into the configuration templates.
